### ISSUE TITLE
SreamVR2.0.1 Too Many Errors!!

### ISSUE THREAD

**Issue by superjayman** [ValveSoftware/steamvr_unity_plugin#135]
What's going on with your latest release on the Asset store, nothing works!! it's poorly documented. I don't even know where to begin, I've encountered so many issues, eg. controllers don't show up, if they do, they don't work in run-time.  Aren't you guys testing your stuff?? and be clear in your explanation of the step we need to take.

**Comment by nebuch** [comment]
Whole controller thing is changed and there is no documentation about it. No one know how to work atm.

**Comment by superjayman** [comment]
Does anyone even respond here? or are they too busy fixing all the bugs?

**Comment by zite** [comment]
Please try to read the pdf documentation included with the plugin. If you are trying to upgrade from an older version of SteamVR significant modifications to your code will be necessary

**Comment by superjayman** [comment]
Hey Zite, read the comments!!! this Release is crap. Some one on Asset store said 


> "I spent days in my unity projects to get this working,but, I don't think it's worth the time because it ends up crasching Unity. I've been testing Unity 2018.2.10 2018.3 b3 and some more versions to be sure.
> 
> When spamming runtime errors, it might be that you only have one controller available. Just put on both controllers to get rid of THAT issue.
> 
> Some other issues can be that you haven't run the script that generates the input files.
> 
> Overall, this could be ok for a alpha branch, not a release branch!
> 
> It's too bad Oculus having the best software developers.
> 
> TLDR  I will roll back to old, boring but working 1.x version again."

You should test it properly don't get people to do your work for you!.. 

Version 2.0 Release is Alpha At Best!! There are many issues too many to list.

**Comment by kisak-valve** [comment]
@superjayman, please try to keep your posts civilized.

**Comment by zite** [comment]
Closing this as I responded to a more recent issue. #150
